stripe/src/main/java/com/stripe/android/view/ShippingMethodAdapter.java

```diff
@@ -0,0 +1,90 @@
+package com.stripe.android.view;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.stripe.android.model.ShippingMethod;
+
+import java.util.List;
+
+/**
+ * Adapter that populates a list with shipping methods
+ */
+class ShippingMethodAdapter extends RecyclerView.Adapter<ShippingMethodAdapter.ViewHolder> {
+
+    private List<ShippingMethod> mShippingMethods;
+    private int mSelectedIndex = 0;
+
+    ShippingMethodAdapter() {
+    }
```

Comment:
Can oneline `() { }` if empty -- though make sure you're not supposed to include a call to super() first.